<!-- -->
<launch>

  <!-- send table.xml to param server -->
  <arg name="range" default="5.0" />
  <arg name="shift" default="1.0" />
  <arg name="use_openmp" default="false" />

  <!-- rosrun ndt_localizer ndt_mapping  -->
  <node pkg="ndt_localizer" type="queue_counter" name="queue_counter" output="log" />
  <node pkg="ndt_localizer" type="ndt_mapping" name="ndt_mapping" output="screen">
  	<param name="range" value="$(arg range)" />
    <param name="shift" value="$(arg shift)" />
    <param name="use_openmp" value="$(arg use_openmp)" />
  </node>
  
</launch>
